packages/react-components/src/trace-explorer/trace-explorer-views-widget.tsx

```diff
@@ -149,6 +149,7 @@ export class ReactAvailableViewsWidget extends React.Component<ReactAvailableVie
 
     protected doExperimentSelected(experiment: Experiment | undefined): void {
         this._selectedExperiment = experiment;
+        this.setState({availableOutputDescriptors: []});
```

Comment:
This method is also called after clicking a view by the method `doHandleOutputClicked` through the  call `signalManager().fireExperimentSelectedSignal(this._selectedExperiment);` This chain of events causes the list to be cleared and refreshed when opening a view. Could you please make sure that the list is not refreshed in that case?